### PR TITLE
Completed html-block-and-inline excercise -Kaitlin Lovrich

### PR DESCRIPTION
**Pull request by kaitlin-lovrich**
image

### CODE REVIEW

File: html-block-and-inline/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (shawnkost) [inline html-block-and-inline/index.html]: Don't forget to put the name of the exercise here

File: html-block-and-inline/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <p>
+        Part of our new Americana collection, this <em>beautiful</em> blue is
+        already a customer favorite!
+      </p>
+      <p>Enjoy complete <strong>Freedom of Movement!</strong></p>
+      <p>
+        Does not slip and slide like a regular blanket and keeps you warm from
+        head to toe.
+      </p>
+    </main>
```
Comment (shawnkost) [inline html-block-and-inline/index.html:24]: Nicely done with implementing the `main` element!

**Review by shawnkost — APPROVED** [review]
👍🏻

File: html-block-and-inline/index.html (hunk 1 above)
Comment (kaitlin-lovrich) [reply html-block-and-inline/index.html]: Updated!

File: html-block-and-inline/index.html (hunk 2 above)
Comment (kaitlin-lovrich) [reply html-block-and-inline/index.html:24]: thanks :3

**Pull request merged**